dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/jaxb/JAXBContextCache.java

```
@@ -0,0 +1,504 @@
+                } catch (Throwable t2) {
+                    //ignore
+	            // Liberty Change begin
+	            if (isLoggableInfo) {
+	               LOG.info("Ignoring throwable t2: " + t2);  
+                    } 
+                    // Liberty Change end	
+                }
+            }
+            if (fact != null) {
+                for (Method m : fact.getMethods()) {
+                    if ("createContext".equals(m.getName())
+                        && m.getParameterTypes().length == 9) {
+                        try {
+                            return (JAXBContext)m.invoke(null,
+                                     classes.toArray(new Class<?>[0]),
+                                     typeRefs,
+                                     map.get(pfx + "subclassReplacements"),
+                                     map.get(pfx + "defaultNamespaceRemap"),
+                                     map.get(pfx + "c14n") == null
+                                         ? Boolean.FALSE
+                                             : map.get(pfx + "c14n"),
+                                     map.get(pfx + "v2.model.annotation.RuntimeAnnotationReader"),
+                                     map.get(pfx + "XmlAccessorFactory") == null
+                                         ? Boolean.FALSE
+                                             : map.get(pfx + "XmlAccessorFactory"),
+                                     map.get(pfx + "treatEverythingNillable") == null
+                                         ? Boolean.FALSE : map.get(pfx + "treatEverythingNillable"),
+                                     map.get("retainReferenceToInfo") == null
+                                         ? Boolean.FALSE : map.get("retainReferenceToInfo"));
+                        } catch (Throwable e) {
+                            //ignore
+	                    // Liberty Change begin
+	                    if (isLoggableInfo) {
```

Since this exception is ignored, this should be logged only in the trace.log. Check and log needs to change to fine